### DISCUSSION TITLE
Phone number format for brazil/pt

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1464] [Q&A]
### Goal

Avaliar a proposta de inserir os parênteses na máscara de validação do componente.
Exemplo: (11) 99999-0000

### Context

Por limitação do componente, não é possível trazer o parênteses na máscara de validação.

### Impact

ND

### Discussion Questions

Seria possível fazer esse ajuste no componente?

**Comment by yceballost** [comment]
Hi Anne,

Please, open a ticket in Jira because this needs changes in an external package that provides all of this information in code.

**Reply by annefranca** [reply]
Ticket aberto: https://jira.tid.es/browse/OBVIVO-2036
